index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+    <header class="headerOne">
+        <h1>Hello Microverse!</h1>
+    </header>
+  <body>
+  </body>
```

Comment:
- [ ] Please, use correct structure of HTML file. You should also close `html` tag. Also, please, insert all tags inside of `body` tag. Your `header` tag is outside of `body` tag.
- You can see this issue information in linter error
<img width="861" alt="Screen Shot 2022-01-06 at 10 46 02" src="https://user-images.githubusercontent.com/49064106/148340815-9e61b78c-f37c-4ae3-acab-c15eba40407f.png">